migrate.go

```diff
@@ -68,12 +68,12 @@ func migrateDB(config map[string]string) {
 			if err != nil || strings.Contains(string(result), "ERROR") {
 				if err == nil {
 					err = fmt.Errorf("\n%s", string(result))
-				} else {
-					// If at any point we fail, log it and break
-					log.Printf("ERROR loading sql migration:%s\n", err)
-					log.Printf("All further migrations cancelled\n\n")
-					break
 				}
+				// If at any point we fail, log it and break
+				log.Printf("ERROR loading sql migration:%s\n", err)
```

Comment:
I think this line only belongs to the else because `err` can be `nil` sometimes.